CodeEdit/Features/StatusBar/Views/StatusBarItems/StatusBarCursorLocationLabel.swift

```diff
@@ -52,6 +61,12 @@ struct StatusBarCursorLocationLabel: View {
             return "\(cursorPositions[0].range.length) characters"
         }
 
+        // When the option key is pressed display the character offset.
+        if modifierKeys.contains(.option) {
+            return "Char: \(cursorPositions[0].range.location) Len: 0"
```

Comment:
Why is Len: 0?

Reply:
It's the case where the range's length is 0 because of the check on line 49

Reply:
It seems that this is due to line 49. It appears that this piece of code is meant to run only when there's just one cursor, not an selection.

Reply:
Ups didn't refresh the page...